specification.md

```diff
@@ -1083,7 +1083,7 @@ format](https://flux-framework.readthedocs.io/projects/flux-rfc/en/latest/spec_1
 A jobspec V1 YAML document SHALL consist of a dictionary defining the
 resources, tasks and other attributes of a single program. The dictionary
 MUST contain the keys `resources`, `tasks`, `attributes`, and `version`.
-Each of the listed jobspec keys SHALL meet the form and requirements
+Each of the listed job specification keys SHALL meet the form and requirements
```

Comment:
Sorry @andre-merzky . I should have been more verbose in my comment.  There are a hand full of these `jobspec` -> `job specification` changes that I think need to be reverted.   I won't tag them all to keep the noise on this PR down, but I count about 8 in total.